mmv1/products/identityplatform/Config.yaml

```diff
@@ -62,6 +62,84 @@ properties:
     name: 'autodeleteAnonymousUsers'
     description: |
       Whether anonymous users will be auto-deleted after a period of 30 days
+  - !ruby/object:Api::Type::NestedObject
+    name: 'signIn'
```

Comment:
I think all fields are mutable.

Reply:
Do you think if it would make sense to add/update some tests to test the update functionality of these fields. More details on update tests: https://googlecloudplatform.github.io/magic-modules/develop/test/#add-an-update-test

Reply:
Looks like this is not addressed. Sorry, if you're still working on it. Absolutely no rush. Just want to make sure that you are aware of this comment. Please let me know if you have any question implementing an update tests.